src/main/java/org/codehaus/mojo/license/AbstractThirdPartyReportMojo.java

```diff
@@ -345,6 +359,28 @@
     // AbstractMavenReport Implementation
     // ----------------------------------------------------------------------
 
+    /**
+     * Method to initialize the mojo before doing any concrete actions.
+     *
+     * <b>Note:</b> The method is invoked before the {@link #executeReport()} method.
+     *
+     * @throws Exception if any
+     */
+    protected void init()
+        throws Exception
+    {
+        if ( licenseMergesUrl != null )
+        {
+            if ( HttpRequester.isStringUrl( licenseMergesUrl ) )
+            {
+                InputStream input = new URL( licenseMergesUrl ).openStream();
```

Comment:
Let's use UrlRequester that is coming via #249 to do this.

Reply:
I'm waiting for merging of #249

Reply:
I just merged #249